### PR TITLE
first cut at getting a data downloader

### PR DESCRIPTION
**Pull request by doutriaux1**
@gleckler1 @durack1 please review @zshaheen you too please if you have time

### CODE REVIEW

File: src/python/misc/scripts/get_pmp_data.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        for chunk in r.iter_content(chunk_size=1024):
+            if chunk:  # filter local_filename keep-alive new chunks
+                f.write(chunk)
+
+
+parser = PMPParser(description='Get sample data')
+parser.add_argument("--dataset", help="Download observation or sample data or both",
+                    default="all", choices=["all", "obs", "sample"])
+parser.add_argument("--version", help="which version to use", default="latest")
+parser.add_argument("--server", help="which serverto use",
```
Comment (durack1) [inline src/python/misc/scripts/get_pmp_data.py]: serverto -> server to

File: src/python/misc/scripts/get_pmp_data.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                    default="https://pcmdiweb.llnl.gov/pss/pmpdata")
+parser.add_argument("--version_in_path", action="store_true", default=False,
+                    help="Append version in rooot path, avoids clobbering versions")
```
Comment (durack1) [inline src/python/misc/scripts/get_pmp_data.py]: Roooooooot -> root

**Comment by durack1** [comment]
@doutriaux1 this is not in nightlies yet is it? Can I conda install this?

**Comment by doutriaux1** [comment]
No worries. I got quite a week as well so I can wait!

**Comment by gleckler1** [comment]
@doutriaux1 easy to use and working well!

**Pull request merged**